graspologic/embed/ase.py

```diff
@@ -58,7 +60,7 @@ class AdjacencySpectralEmbed(BaseEmbed):
         a UserWarning is thrown. Not checking for connectedness may result in
         faster computation.
 
-    diag_aug : bool, optional (default = True)
+    diag_aug : bool, optional (default = False)
```

Comment:
dont know that i agree with this, is a big conversation

Reply:
True. I reverted it in 7338b01. Original reasoning was based around anton's find about diag_aug producing slightly different results between `fit --> transform` and `fit_transform`, but I think the current implementation should circumvent that.